.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -35,56 +35,18 @@ body:
       required: true
   - type: input
```

Comment:
"System Information" should be a textarea too.

Also, inconsistent capitalization if I'm not mistaken:

 - "System Information" => title case
 - "Minetest version", "Steps to reproduce" => sentence case